cmd/panfs.go

```diff
@@ -776,9 +772,8 @@ func (fs *PANFSObjects) DeleteBucket(ctx context.Context, bucket string, opts De
 
 	globalBucketMetadataCache.Delete(bucket)
 	if fs.configAgent != nil {
-		noLockID := ""
-		if err = fs.configAgent.DeleteObject(
-			pathJoin(panfsBucketListPrefix, bucket), noLockID); err != nil {
+		prefix := pathJoin(minioMetaBucket, bucketMetaPrefix, bucket, SlashSeparator)
```

Comment:
again, as we discussed in teams, probably we can simplify this code a bit, like:

```
if fs.configAgent == nil {
   if err = fsRemoveAll( ...
   ...
} // do nothing in case of pan-config, as deleteBucketMetadata(ctx, fs, bucket) will care about everything
```